geoh5py/objects/object_base.py

```diff
@@ -448,6 +465,11 @@ def property_groups(self) -> list[PropertyGroup] | None:
         """
         return self._property_groups
 
+    def post_processing(self):
```

Comment:
is this an abstract class?

Reply:
No it's just there to maintain the older Drillhole object. It just does nothing for everyone else.